### ISSUE TITLE
Suggestion for readme file update

### ISSUE THREAD

**Issue by ryanthompson591** [theaigames/conquest-engine#1]
for linux/mac
replace:
ls _.java > ...
with
find . -type f -name "_.java" > sources.txt

change:
remove the path ending with .sh
to:
remove the .sh strings so your bots look like this:
bot1 = new IORobot(bot1Dir);

**Comment by echnyee** [comment]
Agree, some part of the description in readme is vague and not precise.
by the way, this cmd works for me. just add a star symbol in find cmd. find . -type f -name "*.java" > sources.txt
